schema/deploy/computed_columns/form_change_total_project_value.sql

```diff
@@ -17,25 +13,37 @@ with additional_funding_sources as
         and operation != 'archive'
         )::jsonb
       ) as x(source text, amount numeric, status text)
-    )
-  select
+    ),
+additional_funding_sources_sum as
+    (
+        select sum(amount::numeric) as total from additional_funding_sources where status = 'Approved'
+    ),
+additional_funding_sources_exist as
```

Comment:
What do we want to show for `total_project_value` when an additional_funding_source status is `awaiting_approval` or `denied`?
Currently in both cases `total_project_value` will show the "cannot calculate.." message. Is this what we want?
Scenario:
An additional funding source is denied, until it is removed the `total_project_value` will never calculate. I think at least for `denied` it should return 0 as this may be data that we won't want to remove, but if we don't remove it, total_project_value is permanently unable to calculate.

There is also a bit of a behavioural inconsistency because of the `select 1 from cif.form_change...` block below on line 23. The calculation will work and consider `denied` and `awaiting_approval` as 0 when there is at least 1 approved source in the set, but will not calculate if the set only includes `denied` and `awaiting_approval` sources.

Reply:
As discussed with @gurjmatharu & @dleard:
- As long as there's one or more `awaiting_approval`, `total_project_value` cannot be calculated.
- As long as the rest are `approved`, when there are `denied`, `total_project_value` can be calculated.